src/components/Calculator.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import { Textfit } from 'react-textfit';
```

Comment:
- [ ] I got a compilation error when I try to start the app. Can you please take a look at it? It seems like the problem is coming from `react-textfit`. 

Screenshot |
:--------------:|
![a1](https://user-images.githubusercontent.com/39523312/161291290-667d99df-bd3d-449a-86aa-3b552141e61b.png)

Reply:
Yes it's ok, I added in the PR description the command to run:
Nb: don't forget to run
> npm i react-textfit

Reply:
@Mehdi-Rh I highly suggest to add it to package.json